src/meltano/cli/utils.py

```diff
@@ -415,15 +416,15 @@ def install_status_update(install_state):
         PluginInstallStatus.SKIPPED,
     }:
         msg = f"{install_state.verb} {desc} '{plugin.name}'..."
-        click.secho(msg)
+        logger.info(msg)
```

Comment:
Should this use `%` style interpolation into `logger.info` instead of an f-string? The same comment applies a few lines below.

Reply:
Ah, good catch. Ruff missed this cause it's not a literal.

bfcbd71d9